docs/Create-Packages/Creating-a-Package.md

```diff
@@ -236,22 +236,20 @@ When using a convention-based working directory, on the other hand, NuGet assume
+- PowerShell scripts only initialization script init.ps1 (packages used with NuGet packages.config format can include installation scripts as well, which is not supported in NuGet project.json and packageReference formats).
+- Transformations to source code files in a project via .pp files.
 
 The folder conventions are as follows:
 
 | Folder | Description | Action upon package install |
 | --- | --- | --- |
 | (root) | Location for readme.txt | Visual Studio displays a readme.txt file in the package root when the package is installed. |
-| lib | Assembly (`.dll`), documentation (`.xml`), and symbol (`.pdb`) files | Assemblies are added as references; `.xml` and `.pdb` copied into project folders. See [Supporting multiple target frameworks](Supporting-Multiple-Target-Frameworks.md) for creating framework target-specific sub-folders. |
-| runtimes | Architecture-specific assembly (`.dll`), symbol (`.pdb`), and native resource (`.pri`) files | Assemblies are added as references; other files are copied into project folders. See [Supporting multiple target frameworks](Supporting-Multiple-Target-Frameworks.md). |
-| content | Arbitrary files | Contents are copied to the project root |
-| build | MSBuild `.targets` and `.props` files | Automatically inserted into the project file (NuGet 2.x) or `project.lock.json` (NuGet 3.x+). |
-| tools | Powershell scripts and programs accessible from the Package Manager Console | Contents are copied to the project folder, and the `tools` folder is added to the PATH environment variable. |
-
-Think of the **content** folder as the root of the target application. To have the package add an image in the application's */images* folder, place it in the package's *content/images* folder.
+| lib | Assembly (`.dll`), documentation (`.xml`), and symbol (`.pdb`) files | Assemblies are added as references; `.xml` and `.pdb` are extracted in packages folder. See [Supporting multiple target frameworks](Supporting-Multiple-Target-Frameworks.md) for creating framework target-specific sub-folders. |
+| runtimes | Architecture-specific assembly (`.dll`), symbol (`.pdb`), and native resource (`.pri`) files | Assemblies are added as references; other files are extracted in packages folder. See [Supporting multiple target frameworks](Supporting-Multiple-Target-Frameworks.md). |
+| contentFiles | Arbitrary files | Contents are extracted in packages folder and linked to target project based on buildAction. See [Including content files](../schema/nuspec.md#including-content-files). |
+| build | MSBuild `.targets` and `.props` files | Automatically inserted into the project file (packages.config) or `project.lock.json` (project.json) or (`<projectName>.csproj.nuget.g.targets`/ `<projectName>.csproj.nuget.g.props`). |
+| tools | Powershell scripts and programs accessible from the Package Manager Console | Contents are extracted into the packages folder, and the `tools` folder is added to the PATH environment variable. |
```

Comment:
Add `buildMultitargeting`  this is a build folder that does not use a TFM, it is referenced in the outer build only for multi-framework projects.

Reply:
added